### ISSUE TITLE
Bug: WinUI-2.5: Accessibility: [MAS4.1.1 - Parsing]: Narrator: The keyboard and narrator focus is not in sync using Tab key for the pager control data grid elements

### ISSUE THREAD

**Issue by v-amnat** [microsoft/microsoft-ui-xaml#3633]
**Tags**
Accessibility, A11yFT,  A11yMAS, A11yMediumImpact, MAS4.1.1, Narrator, Sev3, WinUI-2.5, Win32, Wipro 

**User Impact**
The keyboard and narrator focus is not in sync, so the narrator user might face difficulty to traverse the pager control data grid elements using Tab key.

**Repro-Steps**
1. Launch the WinUI_2.5_TestApp
2. Navigate to the Pager control tab
3. Navigate to the Pager control data grid 
4. Turn On the Narrator  traverse the data  grid elements using Tab key from Pager control tab element and observe the keyboard and narrator focus order.

**Expected Behavior**
The keyboard and narrator focus should be in sync using Tab key for the pager control data grid elements

**Actual Behavior**
The keyboard and narrator focus is not in sync using Tab key for the pager control data grid elements

**Repro attachments**
[Narrator-Parsing Issue for Data grid_edit.zip](https://github.com/microsoft/microsoft-ui-xaml/files/5558450/Narrator-Parsing.Issue.for.Data.grid_edit.zip)

**Hardware and build details**
Win32
Windows OS Version: Windows 10
App: WinUI_2.5_TestApp

**Comment by YuliKl** [comment]
Closing this issue as well because the data grid is not a WinUI control. Related to #3632. 

@MarissaMatt FYI in case you decide to use the data grid in the future. I recommend filing a bug on the owners of that control.
